### ISSUE TITLE
PS4 wireless controller Support?

### ISSUE THREAD

**Issue by ActuallyGsman** [carl-anders/slimevr-wrangler#6]
Is PS4/PS4 Wireless Controller support possible?

**Comment by HorrorAssociation10** [comment]
Yeah, what about adding PS Move controllers support? Would be cool, i think
Thanks in advance :)

**Comment by Aravind-S-S** [comment]
Is there support for reading Euler angles from generic controllers(Like Arduino, etc.,) that can stream data via BLE?
